content/material_and_questions.md

```diff
@@ -1,5 +1,11 @@
 # Material and Questions
 
+Version
```

Comment:
I decided to define Version here. I left out this part, because it either (a) doesn't belong here, or (b) is redundant
"For the International Finals, questions must be written covering the entire material for the quiz year.
Questions for the International Finals will be the responsibility of the CQLT.  The head answer judge is responsible for the correctness of questions.
Questions must be word-for-word as it appears in the 2011 NIV."

Reply:
I think I agree. Just to be clear: Version should be defined somewhere in the rulebook, and here seems like a good place, but the whole "for IBQ" stuff doesn't belong in the rulebook proper.